libioc/Jail.py

```diff
@@ -1816,22 +1817,21 @@ def _apply_resource_limits(
                 continue
 
             rule = f"jail:{self.identifier}:{key}:{rlimit_prop.limit_string}"
-            args.append("-a")
-            args.append(rule)
+            try:
+                libioc.helpers.exec(
+                    ["/usr/bin/rctl", "-a", rule],
+                    logger=self.logger
+                )
+                skipped = False
+            except Exception:
+                yield event.fail()
+                raise
 
-        if len(args) == 0:
+        if skipped is True:
```

Comment:
how would this be true?

Reply:
When no resource limit was applied, this value stays True, so that the event can be properly flagged as skipped.

Reply:
are you sure we didn't make a mistake of some sort in that case?

Reply:
No, because Resource Limits are treated as skipped until at least one was applied successfully. Anyhow the following `return` can be removed, which is done in the latest commit.